ui/src/app/base/components/Pager/Pager.test.js

```diff
@@ -0,0 +1,94 @@
+import { mount } from "enzyme";
+import React from "react";
+
+import Pager from "./Pager";
+
+describe("<Pager />", () => {
+  // snapshot tests
+  it("renders and matches the snapshot", () => {
+    const component = mount(
```

Comment:
I think these can all be shallow.

Reply:
Quite right, thanks, just had to change an assertion for the rendered class to `PagerItem`.